src/schemathesis/specs/openapi/negative.py

```diff
@@ -0,0 +1,65 @@
+from copy import deepcopy
+from typing import Any, Callable, Dict, TypeVar
+
+from hypothesis import strategies as st
+from hypothesis.strategies._internal.featureflags import FeatureStrategy
+from hypothesis_jsonschema import from_schema
+
+T = TypeVar("T")
+Draw = Callable[[st.SearchStrategy[T]], T]
+Schema = Dict[str, Any]
+# draw, new_schema, property schema, name
+Mutation = Callable[[Draw, Schema, Schema, str], None]
+
+
+def negative_schema(schema: Dict[str, Any]) -> st.SearchStrategy:
+    """A strategy for objects, that DO NOT match the input schema."""
+
+    # First we mutate the input schema, so any valid instance for it is invalid to the original schema
```

Comment:
This makes sense; but note that one option which might be easier to implement in general (e.g. absence of a required `""` key when random extra properties are allowed) would be to calculate a negative schema with *little* overlap and then filter by "instance is not valid against original schema".

Reply:
Interesting, I will try that! :)

Reply:
I added `{"not": schema}` shortcut for schemas where the best way to negate is not clear - specific handling for such schemas can be added later. Also, now the output is filtered with `not validator.is_valid(s)`, which basically should work as you mention :) Then in mutations, there could be overlap in the resulting schema, but in the end, it will be filtered out :)